cli/command_acl.go

```diff
@@ -7,11 +7,11 @@ type commandServerACL struct {
 	list   commandACLList
 }
 
-func (c *commandServerACL) setup(parent commandParent) {
+func (c *commandServerACL) setup(app appServices, parent commandParent) {
```

Comment:
Could `app` be instead `as` or `svc`, since it is not the app, but instead the services used by the app?

Reply:
renamed to `svc` everywhere